rllib/models/specs/tests/test_spec_dict.py

```diff
@@ -0,0 +1,189 @@
+import unittest
+import numpy as np
+
+from ray.rllib.models.specs.specs_np import NPTensorSpec
+from ray.rllib.models.specs.specs_dict import SpecDict
+from ray.rllib.models.specs.checker import check_input_specs
+
+
+class TypeClass1:
+    pass
+
+
+class TypeClass2:
+    pass
+
+
+class TestSpecDict(unittest.TestCase):
+    def test_basic_validation(self):
+
```

Comment:
Can you add a single line """-comment below each test case as to what aspect of Specs we are testing with these?

Reply:
done.